Add repo check script

cnealen opened the pull request:
## Summary
- Add `repo_check.sh` script to monitor status of NGWPC repositories
- Script checks latest commits, timestamps, and GitHub Actions workflow runs across multiple repos
- Supports GitHub token authentication for higher API rate limits

## Features
- Monitors 5 NGWPC repositories (ngencerf-server, ngen, ngen-forcing, nwm-cal-mgr, nwm-fcst-mgr)
- Displays latest commit information with human-readable timestamps
- Shows last 5 workflow runs with status indicators

cnealen committed:
feat: add repo checker script

cnealen committed:
adding untracked files

cnealen committed:
added UI

cnealen committed:
re-ordered to match build order

miguelp1986 reviewed (commented):
@cnealen we probably want to put these files within a directory. Maybe a 'utils' directory.

miguelp1986 commented:
It would be helpful to know what kind of event a workflow run is for. If you click on Event you can see the options there (pull_request, push, release). Knowing which kind of event a workflow run let's us know what triggered it so we can know if we're looking at a push to development, for example.


<img width="1099" height="470" alt="Screenshot 2026-01-29 at 11 33 36" src="https://github.com/user-attachments/assets/dd497668-c926-4751-b0ba-578594be8d82" />

cnealen committed:
Header row now includes an Event column
Each workflow run now displays its event type (e.g., pull_request, push, release) pulled from the API's event field
Both are padded/aligned to keep the table readable

miguelp1986 reviewed (changes requested):
@cnealen I don't think we want to start keeping files like `.claude/plan.md` around in this repo. I'd like to keep it lean and only what we need to run our scripts. Please remove it. Same with `test_dependency_prompts.sh`. That is just a collection of echo commands. Seems like it's probably more appropriate for a README. Perhaps you can consolidate it with `TEST_PLAN.md`? Please also remove `CODE_VERIFICATION.md`.

cnealen committed:
removed unwanted files
